tests/integration/ambient/bookinfo_test.go

```diff
@@ -107,7 +107,58 @@ func TestBookinfo(t *testing.T) {
 					})
 				})
 			})
-
+			// pod off ambient mode
+			nsConfig.RemoveLabel(constants.DataplaneMode)
```

Comment:
Thank you for the test - minor nit/request: it would be good to also have a test where we create an entirely new, never-labeled namespace, deploy the workload, then enroll that namepace for the first time and test again, rather than only doing what this test does, which is label, unlabel, deploy and relabel.

it _should not_ matter but it is a slightly different event flow in the CNI and it's the flow that our docs will use, so ideally I'd like to see it tested both ways just to be safe.